.github/workflows/static-checks-mypy-docs.yml

```diff
@@ -243,7 +243,9 @@ jobs:
       INCLUDE_SUCCESS_OUTPUTS: "${{ inputs.include-success-outputs }}"
       PYTHON_MAJOR_MINOR_VERSION: "${{ inputs.default-python-version }}"
       VERBOSE: "true"
-    if: inputs.canary-run == 'true' && inputs.branch == 'main'
+      # Temporary disabled it until self-hosted ASF runners will be working again
```

Comment:
```suggestion
    # Temporary disabled it until self-hosted ASF runners will be working again
```

Reply:
Won't pass linter I am afraid - it expect commment at the level of next element in yaml

Reply:
Or previous :).

Reply:
oh geez, gotcha.